### PR TITLE
EN-53806 Update OpenDataNetwork.com footer

### PR DESCRIPTION
**Pull request by karrysa**
Plus merge 14 other commits from staging to master. @saraid Are your changes safe to merge to master?

### CODE REVIEW

**Comment by saraid** [comment]
My changes should be in production already, so it should be fine to merge them.

File: gulpfile.js [hunk 1]
```diff
@@ -62,8 +62,7 @@ var searchScripts = baseScripts
         'src/infinite-scroll.js',
         'src/show-more-questions.js',
         'src/mobile-menus.js',
-        'src/search.js',
-        'src/api/odn-api.js']);
```
Comment (tkbrigham) [inline gulpfile.js:66]: Is this intentionally removed?

Comment (tkbrigham) [reply gulpfile.js:66]: Hmmm, looks like the file doesn't exist.

Comment (karrysa) [reply gulpfile.js:66]: See the "Security Patches" change above. Notes say:
_gulp upgraded to 4.0.2, with changes to gulpfile.js src/api/odn-api.js is a non-existent file._

Comment (karrysa) [reply gulpfile.js:66]: @haibeeb Are your changes safe to merge to master?

**Comment by karrysa** [comment]
@haibeeb Are your changes safe to merge to master?

**Review by progman32 — APPROVED** [review]
> My changes should be in production already, so it should be fine to merge them.

It appears that https://github.com/socrata/opendatanetwork.com/commit/e623393d7d2ceff0f94c1950310881245098e80c is currently running in production. This is Manoj's changes. It appears @saraid 's changes were previously deployed, but were overwritten in production when we deployed Manoj's changes. Let's get this merged and everyone's changes re-deployed properly.

**Pull request merged**